apps/common/views/partials/confirmation-banner.html

```diff
@@ -0,0 +1,10 @@
+<div class="alert-complete" role="alert">
+  <span class="icon icon-complete" role="presentation"></span>
+  <div class="alert-message">
+    <h1>{{#t}}pages.confirmation.header{{/t}}</h1>
+
+    {{#values.email}}
+      <p>{{#t}}pages.confirmation.email{{/t}} <strong>{{values.email}}</strong></p>
```

Comment:
This <p> element has been moved inside the <div>. Provided that is deliberate, this PR LGTM.